### DISCUSSION TITLE
现在能进行日语的tts吗？或者说，有计划推进日语tts吗？

### DISCUSSION THREAD

**Discussion by fengyinhuaxu** [lenML/Speech-AI-Forge#159] [问答 | Q&A]
如题，毕竟现在在配音行业中，日配才是标杆

**Comment by zhzLuke96** [comment]
cosyvoice支持日语，直接填日语，或者用 `<|jp|>` 标志日语开始

chattts不支持，具体支不支持这也是上游决定的事情， forge 这里不会特别进行开发
